internal/kafka/internal/services/cloud_providers_test.go

```diff
@@ -145,10 +145,23 @@ func Test_CloudProvider_ListCloudProviders(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "failed to get provider implementation",
```

Comment:
The description should be something along the lines of:

An error is returned when the cloud provider cannot be obtained